### ISSUE TITLE
[BUG] vtu export renumbers pointIDs

### ISSUE THREAD

**Issue by RolBurgHxgn** [nschloe/meshio#1407]
Hi!

(Not sure if this is a bug or a feature request...)

I am using meshio python module to convert nastran .bdf files to .vtu files. Paraview is used to read the vtu files and I figured out that at some point the node IDs were renumbered. However, for many work flows it is important that the points numbering does not change. It seems to me that the renumbering occurs in the mesh.write() process. Paraview itself claims to not renumber anything, see https://discourse.paraview.org/t/option-to-disable-point-and-cell-renumbering/11792

Basically I do:
read bdf file: mesh = meshio.read(bdf_input_file) 
add some point data: mesh.point_data["dT"]=dt
export vtu file: mesh.write("mesh.vtu")

After reading the mesh the point ids I get with mesh.points_id are "correct". 

So the question is: does meshio renumber during .vtu export? If yes, can it be disabled?

Best,
Roland
